back/CourseSheetModule/CourseSheetModel.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Doesn't respect the PSRs
No namespace
Error in the require_once
CourseSheetModel must not extend Connection